doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -0,0 +1,1545 @@
+    media_side_input_loopback_supported         = BOOLEAN                          ; media input loopback support
+    media_side_output_loopback_supported        = BOOLEAN                          ; media output loopback support
+    media_output_loopback                       = BOOLEAN                          ; media side output loopback enable
+    media_input_loopback                        = BOOLEAN                          ; media side input loopback enable
+    host_output_loopback_lane1                  = BOOLEAN                          ; host side output loopback enable lane1
+    host_output_loopback_lane2                  = BOOLEAN                          ; host side output loopback enable lane2
+    host_output_loopback_lane3                  = BOOLEAN                          ; host side output loopback enable lane3
+    host_output_loopback_lane4                  = BOOLEAN                          ; host side output loopback enable lane4
+    host_output_loopback_lane5                  = BOOLEAN                          ; host side output loopback enable lane5
+    host_output_loopback_lane6                  = BOOLEAN                          ; host side output loopback enable lane6
+    host_output_loopback_lane7                  = BOOLEAN                          ; host side output loopback enable lane7
+    host_output_loopback_lane8                  = BOOLEAN                          ; host side output loopback enable lane8
+    host_input_loopback_lane1                   = BOOLEAN                          ; host side input loopback enable lane1
+    host_input_loopback_lane2                   = BOOLEAN                          ; host side input loopback enable lane2
+    host_input_loopback_lane3                   = BOOLEAN                          ; host side input loopback enable lane3
+    host_input_loopback_lane4                   = BOOLEAN                          ; host side input loopback enable lane4
+    host_input_loopback_lane5                   = BOOLEAN                          ; host side input loopback enable lane5
+    host_input_loopback_lane6                   = BOOLEAN                          ; host side input loopback enable lane6
+    host_input_loopback_lane7                   = BOOLEAN                          ; host side input loopback enable lane7
+    host_input_loopback_lane8                   = BOOLEAN                          ; host side input loopback enable lane8
+
+
+#### 2.2 Show interfaces transceiver CLI
+Displays diagnostic monitoring information of the transceivers
+
+**show interfaces transceiver**
+
+This command displays information for all the interfaces for the transceiver requested or a specific interface if the optional "interface_name" is specified.
+
+- Usage:
+  ```
+  show interfaces transceiver (info | eeprom [-d|--dom] | presence | status | pm | loopback) [<interface_name>]
```

Comment:
Comment from @longhuan-cisco (Longyin)
	1.	Propose to have “show interfaces transceiver dom” sub-command for TRANSCEIVER_DOM_SENSOR table info and keep "show interfaces transceiver eeprom [-d|--dom]" as it was for now (for backward compatibility).
	2.	I didn't see a CLI for dom threshold info. Shall we have one? Something like “show interfaces transceiver threshold” for 2.1.3 Transceiver DOM Threshold Table

Reply:
1. Modified in 10/04 commit
2. I will add a CLI for threshold info. Working on that. - 10/04/2022

Reply:
@qinchuanares 
IMO, in general, a -- or - or hyphen characters to be avoided in a CLI.
Hence, the proposal was to remove these characters and just have sub-command ('dom'). 
IMO, this should be looked into as change/enhancement.
I'd suggest put this as change/enhancement request in this HLD and raise a git issue